### PR TITLE
Add blast recipes

### LINKED ISSUE #146: Blast Furnace compatibility for the ore chunk
**Is your feature request related to a problem? Please describe.**
Since vanilla added a new kind of specialization furnace that specialize in speeding up ore to ingot furnace, it's been a time saver and great for early fast smelting before you get into other tech mod  
**Describe the solution you'd like**
It would be cool to allows the ore chunk to be allowed to be smelted in the blast furnace like a vanilla ore does

### PR DESCRIPTION
**Pull request by remplerus**
Fix #146

### CODE REVIEW

**Comment by DreaminBoi** [comment]
nice ty!

**Review by NovaMachina — CHANGES_REQUESTED** [review]
Please include recipes in the data generator.

**Review by remplerus — COMMENTED** [review]
added data gen

File: src/main/java/novamachina/exnihilosequentia/api/datagen/AbstractRecipeGenerator.java [hunk 1]
```diff
@@ -73,6 +73,10 @@ protected void registerSmelting(EnumOre ore, Consumer<IFinishedRecipe> consumer)
                 .smeltingRecipe(Ingredient.fromItems(ore.getChunkItem().get()), ore.getIngotItem() != null ? ore.getIngotItem() : ore.getIngotRegistryItem().get(), 0.7F, 200)
                 .addCriterion(CHUNK_CONDITION, InventoryChangeTrigger.Instance.forItems(ore.getChunkItem().get()))
                 .build(consumer, new ResourceLocation(modId, prependRecipePrefix(ore.getIngotName())));
+        CookingRecipeBuilder
+                .blastingRecipe(Ingredient.fromItems(ore.getChunkItem().get()), ore.getIngotItem() != null ? ore.getIngotItem() : ore.getIngotRegistryItem().get(), 0.7F, 200)
+                .addCriterion(CHUNK_CONDITION, InventoryChangeTrigger.Instance.forItems(ore.getChunkItem().get()))
+                .build(consumer, new ResourceLocation(modId, prependRecipePrefix(ore.getIngotName())));
```
Comment (NovaMachina) [inline src/main/java/novamachina/exnihilosequentia/api/datagen/AbstractRecipeGenerator.java]: Duplicate recipe name

File: src/main/java/novamachina/exnihilosequentia/common/datagen/ExNihiloRecipeGenerator.java [hunk 2]
```diff
@@ -680,13 +681,23 @@ private void registerOres(Consumer<IFinishedRecipe> consumer) {
                             .addCriterion(CHUNK_CONDITION, InventoryChangeTrigger.Instance
                                     .forItems(ore.getChunkItem().get()))
                             .build(consumer, createSaveLocation(new ResourceLocation(ExNihiloConstants.ModIds.MINECRAFT, "ingot_iron")));
+                    CookingRecipeBuilder
+                            .blastingRecipe(Ingredient.fromItems(ore.getChunkItem().get()), Items.IRON_INGOT, 0.7F, 200)
+                            .addCriterion(CHUNK_CONDITION, InventoryChangeTrigger.Instance
+                                    .forItems(ore.getChunkItem().get()))
+                            .build(consumer, createSaveLocation(new ResourceLocation(ExNihiloConstants.ModIds.MINECRAFT, "ingot_iron")));
```
Comment (NovaMachina) [inline src/main/java/novamachina/exnihilosequentia/common/datagen/ExNihiloRecipeGenerator.java]: Duplicate recipe name

File: src/main/java/novamachina/exnihilosequentia/common/datagen/ExNihiloRecipeGenerator.java [hunk 3]
```diff
@@ -680,13 +681,23 @@ private void registerOres(Consumer<IFinishedRecipe> consumer) {
...
                     CookingRecipeBuilder
                             .smeltingRecipe(Ingredient.fromItems(ore.getChunkItem().get()), Items.GOLD_INGOT, 0.7F, 200)
                             .addCriterion(CHUNK_CONDITION, InventoryChangeTrigger.Instance
                                     .forItems(ore.getChunkItem().get()))
                             .build(consumer, createSaveLocation(new ResourceLocation(ExNihiloConstants.ModIds.MINECRAFT, "ingot_gold")));
+                    CookingRecipeBuilder
+                            .blastingRecipe(Ingredient.fromItems(ore.getChunkItem().get()), Items.GOLD_INGOT, 0.7F, 200)
+                            .addCriterion(CHUNK_CONDITION, InventoryChangeTrigger.Instance
+                                    .forItems(ore.getChunkItem().get()))
+                            .build(consumer, createSaveLocation(new ResourceLocation(ExNihiloConstants.ModIds.MINECRAFT, "ingot_gold")));
```
Comment (NovaMachina) [inline src/main/java/novamachina/exnihilosequentia/common/datagen/ExNihiloRecipeGenerator.java]: Duplicate recipe name

**Review by NovaMachina — APPROVED** [review]
This is really good. Duplicate recipe names aren't allowed as they will override each other.

**Review by NovaMachina — CHANGES_REQUESTED** [review]
Just need to remove the duplicate names

**Review by NovaMachina — APPROVED** [review]

**Pull request merged**